lms/static/js/ccx/schedule.js

```diff
@@ -442,7 +489,18 @@ var edx = edx || {};
 	      });
 	    };
 	},
-
+	changeUnitsDate: function(sequential, date_type, date) {
+	    // This code iterates the children (aka verticals) of a sequential.
+	    // It updates start and due dates to corresponding dates
+	    // of sequential (parent).
```

Comment:
This comment helps a lot, thank you. I would recommend splitting this function into two: `updateStartDate()` and `updateDueDate()`, similarly to how I suggested splitting the `disabled_fields()` function into ` enableFields()` and `disableFields()`.

However, looking at how this `changeUnitsDate()` function is actually called in the codebase, it looks like there is a `what` variable that is passed around and used for one of the parameters, and I don't really understand what `what` is, how it's set, or why it's necessary. It seems like getting rid of that variable would require a larger refactor, which might be out of scope for this pull request.

I'd encourage you to address this in a follow-on pull request. The comment helps a *lot*, but the name `changeUnitsDate()` is still confusing.